### DISCUSSION TITLE
Issue on Stop insert/update operation using set up boot hook on Model

### DISCUSSION THREAD

**Discussion by NeguinhoAdmin** [Laravel-Backpack/community-forum#1081] [General] [answered]
I have set up event hook on model (boot) while record is saving, and I am validating some information from from thirdparty api, such as Valid/Exists Number Plate of vehicle.

upon validating at some point i need to handle gracefully error message and stop 'saving operation'  which pretty this line do -> 
```
session()->flash('error', 'Invalid vehicle registration number.');
                    
                    return false;
```

but on frontend it show green nice message that successfully record is saved while record flushed and discard. 

In there any way to stop and being not discard whole form user filled already, also user to save it again after valid registration number.

```

    protected static function booted()
    {
        static::saving(function ($booking) {

            .....

                } else {
                    Log::info("IT NOT SUCCEEDS");

                    $errors = json_decode($response->body())->errors;

                    foreach ($errors as $error) {
                        Log::error("DVLA API Error: " . $error->detail);
                    }

                    session()->flash('error', 'Invalid vehicle registration number.');
                    
                    return false;
                }
            }

            $booking->title = $booking->status . " MOT " . $booking->vehicle_registration . " " . $booking->customer_name;

            \App::make(\App\Http\Controllers\Admin\MOTBookingCrudController::class)
                ->generateAgreementAccess($booking);
        });
    }
```

**Comment by karandatwani92** [comment, marked as answer]
Hey @NeguinhoAdmin 

You just said what it is. It is validation, So...
I recommend you create [custom validation rule](https://laravel.com/docs/11.x/validation#custom-validation-rules) and then you are free to use it for any request in your app.

**Reply by NeguinhoAdmin** [reply]
Nice that solved
